GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifiableAccountDetail.m

```diff
@@ -17,4 +17,25 @@
 #import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h"
 
 @implementation GIDVerifiableAccountDetail
+
+- (instancetype)initWithAccountDetailType:(GIDAccountDetailType)accountDetailType {
+   self = [super init];
+   if (self) {
+       _accountDetailType = accountDetailType;
+   }
+   return self;
+}
+
+// Returns dictionary mapping account detail type to scope.
++ (NSDictionary<NSNumber *, NSString *> *)scopeMapping {
+  return @{
+    @(GIDAccountDetailTypeAgeOver18) : @"https://www.googleapis.com/auth/verified.age.over18.standard"
+  };
+}
+
+- (NSString *)retrieveScope {
```

Comment:
Alternative implementation to consider:

```
- (NSString *)scope {
  switch (self.accountDetailType) {
    case GIDAccountDetailTypeAgeOver18:
      return kGIDAccountDetailTypeAgeOver18Scope;
    default:
      return nil;
  }
}
```

I think this implementation is a little cleaner since we don't need an auxiliary method to store and create a dictionary every time we want the scope string -- IMHO the enumeration we have already does that.

Reply:
Ah, I see what you mean, also this is definitely simpler so I've swapped. Done.